pkg/cmd/cli/restore/describe.go

```diff
@@ -17,18 +17,186 @@ limitations under the License.
+
 	c := &cobra.Command{
-		Use:   "describe",
-		Short: "Describe a backup",
+		Use:   use,
+		Short: "Describe restores",
 		Run: func(c *cobra.Command, args []string) {
+			arkClient, err := f.Client()
+			cmd.CheckError(err)
+
+			var restores *api.RestoreList
+			if len(args) > 0 {
+				restores = new(api.RestoreList)
+				for _, name := range args {
+					restore, err := arkClient.Ark().Restores(api.DefaultNamespace).Get(name, metav1.GetOptions{})
+					cmd.CheckError(err)
+					restores.Items = append(restores.Items, *restore)
+				}
+			} else {
+				restores, err = arkClient.ArkV1().Restores(api.DefaultNamespace).List(listOptions)
+				cmd.CheckError(err)
+			}
+
+			first := true
+			for _, restore := range restores.Items {
+				s := output.Describe(func(out io.Writer) {
+					describeRestore(out, &restore, arkClient)
+				})
+				if first {
+					first = false
+					fmt.Print(s)
+				} else {
+					fmt.Printf("\n\n%s", s)
+				}
+			}
+			cmd.CheckError(err)
 		},
 	}
 
+	c.Flags().StringVarP(&listOptions.LabelSelector, "selector", "l", listOptions.LabelSelector, "only show items matching this label selector")
+
+	output.BindFlags(c.Flags())
+
 	return c
 }
+
+func describeRestore(out io.Writer, restore *api.Restore, arkClient clientset.Interface) {
+	output.DescribeMetadata(out, restore.ObjectMeta)
+
+	fmt.Fprintln(out)
+	fmt.Fprintf(out, "Backup:\t%s\n", restore.Spec.BackupName)
+
+	fmt.Fprintln(out)
+	fmt.Fprintf(out, "Namespaces:\n")
+	var s string
+	if len(restore.Spec.IncludedNamespaces) == 0 {
+		s = "*"
+	} else {
+		s = strings.Join(restore.Spec.IncludedNamespaces, ", ")
+	}
+	fmt.Fprintf(out, "\tIncluded:\t%s\n", s)
+	if len(restore.Spec.ExcludedNamespaces) == 0 {
+		s = "<none>"
+	} else {
+		s = strings.Join(restore.Spec.ExcludedNamespaces, ", ")
+	}
+	fmt.Fprintf(out, "\tExcluded:\t%s\n", s)
+
+	fmt.Fprintln(out)
+	fmt.Fprintf(out, "Resources:\n")
+	if len(restore.Spec.IncludedResources) == 0 {
+		s = "*"
+	} else {
+		s = strings.Join(restore.Spec.IncludedResources, ", ")
+	}
+	fmt.Fprintf(out, "\tIncluded:\t%s\n", s)
+	if len(restore.Spec.ExcludedResources) == 0 {
+		s = "<none>"
+	} else {
+		s = strings.Join(restore.Spec.ExcludedResources, ", ")
+	}
+	fmt.Fprintf(out, "\tExcluded:\t%s\n", s)
+	var clusterResources string
+	switch {
+	case restore.Spec.IncludeClusterResources == nil:
+		clusterResources = "auto"
+	case *restore.Spec.IncludeClusterResources == true:
+		clusterResources = "included"
+	case *restore.Spec.IncludeClusterResources == false:
+		clusterResources = "excluded"
+	}
+	fmt.Fprintf(out, "\tCluster-scoped:\t%s\n", clusterResources)
+
+	fmt.Fprintln(out)
+	output.DescribeMap(out, "Namespace mappings", restore.Spec.NamespaceMapping)
+
+	fmt.Fprintln(out)
+	s = "<none>"
+	if restore.Spec.LabelSelector != nil {
+		s = metav1.FormatLabelSelector(restore.Spec.LabelSelector)
+	}
+	fmt.Fprintf(out, "Label selector:\t%s\n", s)
+
+	fmt.Fprintln(out)
+	b := true
+	if restore.Spec.RestorePVs != nil {
```

Comment:
probably best to do auto/yes/no printing here like with IncludeClusterResources

Reply:
fixing